src/components/SearchModal/CurrencySearch.tsx

```diff
@@ -255,18 +398,41 @@ export function CurrencySearch({
           </Text>
           <CloseIcon onClick={onDismiss} />
         </RowBetween>
-        <SearchInput
-          type="text"
-          id="token-search-input"
-          placeholder={t`Search token name, symbol or address`}
-          value={searchQuery}
-          ref={inputRef as RefObject<HTMLInputElement>}
-          onChange={handleInput}
-          onKeyDown={handleEnter}
-          autoComplete="off"
-        />
+        <Text style={{ color: theme.subText, fontSize: 12 }}>
+          <Trans>
+            You can search and select <span style={{ color: theme.text }}>any token</span> on KyberSwap
+          </Trans>
+        </Text>
+
+        <SearchWrapper>
+          <SearchInput
+            type="text"
+            id="token-search-input"
+            placeholder={t`Search by token name, token symbol or address`}
+            value={searchQuery}
+            ref={inputRef}
+            onChange={handleInput}
+            onKeyDown={handleEnter}
+            autoComplete="off"
+          />
+          <SearchIcon size={18} color={theme.border} />
+        </SearchWrapper>
+
         {showCommonBases && (
-          <CommonBases chainId={chainId} onSelect={handleCurrencySelect} selectedCurrency={selectedCurrency} />
+          <CommonBases
+            chainId={chainId}
+            tokens={filteredCommonTokens}
+            handleToggleFavorite={handleClickFavorite}
+            onSelect={handleCurrencySelect}
+            selectedCurrency={selectedCurrency}
+          />
+        )}
+        {loadingCommon && (
+          <Flex justifyContent={'center'}>
+            <Text fontSize={12} color={theme.subText}>
+              Loading ...
+            </Text>
+          </Flex>
```

Comment:
Should remove. No need to show `Loading ...` here

Reply:
we need. when:
- Sometimes defaultTokens load slow. 
- an address not in whitelist

Reply:
This is for common base section, not main token section. I dont think we need `Loading ...` for common base since it just an additional part, not main part need user to focus on. We already have loading indicator for main token section.

Reply:
I think it's nice to have. But with the update to favourite tokens instead of common base, should it just be loaded directly from cache so there won't be a slow load case?

Reply:
we save address on local. not save entire token data, so sometime we need call API,

Reply:
I actual faced it and see it's very weird

Reply:
> I think it's nice to have. 

Let's respect our designer =))

Reply:
:tien-sword: no